apps/src/lib/kits/maker/BoardController.js

```diff
@@ -166,13 +166,14 @@ export default class BoardController {
     component.on(event, callback);
   }
 
-  connect() {
-    return BoardController.getDevicePort().then(port => this.connectToBoard(port));
+  static connect() {
+    return BoardController.getDevicePortName()
+        .then(portName => BoardController.connectToBoard(portName));
```

Comment:
Nit: Now that connectToBoard is a static, I think we could just do `BoardController.getDevicePortName().then(BoardController.connectToBoard)`